include/openssl/cms.h

```diff
@@ -491,14 +491,14 @@ int ERR_load_CMS_strings(void);
 # define CMS_R_TYPE_NOT_ENVELOPED_DATA                    146
 # define CMS_R_UNABLE_TO_FINALIZE_CONTEXT                 147
 # define CMS_R_UNKNOWN_CIPHER                             148
-# define CMS_R_UNKNOWN_DIGEST_ALGORIHM                    149
+# define CMS_R_UNKNOWN_DIGEST_ALGORITHM                    149
```

Comment:
My tool is me, a /* */ comment would be enough for me to leave something alone. I can certainly add in the compat item here.